lte/gateway/docker/s1ap/docker-compose.s1ap.yaml

```diff
@@ -0,0 +1,20 @@
+---
+version: "3.7"
+
+services:
+  s1aptester:
+    volumes:
+      - /opt/magma/lte/gateway/configs/:/etc/magma
+    image: ${DOCKER_REGISTRY}agw_s1aptester:${IMAGE_VERSION}
+    container_name: s1aptester
+    command: /bin/bash
```

Comment:
why is the command overwritten here?I thought that container should do something instead of launching a bash?

Reply:
one is expected to directly attach to the container and run commands in it, see https://github.com/magma/magma/pull/11308/files#diff-eab955cb932f81985a4edf889fca5daa713d583b539bf00dd4dc8b024e9eb246R151